app/dao/fact_notification_status_dao.py

```diff
@@ -337,16 +337,15 @@ def get_last_send_for_api_key(api_key_id):
     notification_table = (
         db.session.query(func.max(Notification.created_at).label("last_notification_created"))
         .filter(Notification.api_key_id == api_key_id)
-        .group_by(Notification.api_key_id)
         .all()
     )
-    if not notification_table:
-        return (
+    if not notification_table[0][0]:
```

Comment:
if we left the group by the result would be [], as we are removing the group by as it is redundant - we needed to add this change. Yay for unit tests tha caught this